benches/persistence.rs

```diff
@@ -0,0 +1,167 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+
+use dbsp::trace::ord as dram_ord;
+use dbsp::trace::ord::persistent as persistent_ord;
+use dbsp::trace::BatchReader;
+use dbsp::trace::Builder;
+use dbsp::trace::Cursor;
+use rand::prelude::Distribution;
+use rand::Rng;
+use zipf::ZipfDistribution;
+
+/// What different sizes we should instantiate for the set.
+static BENCHMARK_INPUTS: [u64; 2] = [1 << 12, 1 << 24];
+
+/// Generate a random sequence of operations
+///
+/// # Arguments
+///  - `nop`: Number of operations to generate
+///  - `write`: true will Put, false will generate Get sequences
+///  - `span`: Maximum key
+///  - `distribution`: Supported distribution 'uniform' or 'skewed'
+pub fn generate_operations(nop: usize, span: usize, distribution: &'static str) -> Vec<u64> {
+    assert!(distribution == "skewed" || distribution == "uniform");
+
+    let mut ops = Vec::with_capacity(nop);
+
+    let skewed = distribution == "skewed";
+    let mut t_rng = rand::thread_rng();
```

Comment:
We shouldn't be using a real rng here, we should be using something like [`rand_xoshiro`](https://docs.rs/rand_xoshiro/latest/rand_xoshiro/index.html) so that results are deterministic